weathersimulator-python/func.py

```diff
@@ -0,0 +1,33 @@
+import json
+import os
+import random
+from flask import abort, Flask, jsonify
+from flask_cors import CORS
+from markupsafe import escape
+
+app = Flask(__name__)
+CORS(app)
+
+
+DATA_FOLDER = escape(os.environ.get('DATA_FOLDER', 'data'))
+ERROR_RESPONSE = int(os.environ.get('ERROR_RESPONSE', 404))
+
+
+@app.errorhandler(404)
+def topicNotFound(error):
```

Comment:
Some notes:

- You might want to transform the functions into snake case to be more Python friendly. The original app doesn't follow Python standards.. (my fault.. :rofl:)
- The original app was created to having multiple and different topics (different in theme, like news, finance, etc.). Perhaps in this case, since we are tied to weather we can rename the concept of "topic" to something else. Perhaps something like "data_file" or similar. If we change the usage of "topic" we need to rename the function, the message returned by this function & the parameter in the following function

Reply:
Acknowledged the above mentioned suggestions and have made respective changes in the code.

So I replaced the topic to city and made all the relevant changes in the code.